ext/qiita_marker/qfm_custom_block.c

```diff
@@ -81,26 +53,19 @@ static void free_node_qfm_custom_block(cmark_mem *mem, void *ptr) {
   node_qfm_custom_block *cb = (node_qfm_custom_block *)ptr;
 
   cmark_chunk_free(mem, &cb->info);
-  cmark_strbuf_free(cb->xml_attr_buff);
   mem->free(cb);
 }
 
 static int matches(cmark_syntax_extension *self, cmark_parser *parser,
                    unsigned char *input, int len,
                    cmark_node *parent_container) {
-  int res = 0;
-
-  if (get_qfm_custom_block_opening(parent_container)) {
```

Comment:
Is it possible to contain a custom block as a child when another custom block is the parent?

Reply:
Now, a custom block can contain another custom block, but this is not a specification and may change in the future.